### ISSUE TITLE
Query builder, sql advanced mode pagination

### ISSUE THREAD

**Issue by JuanCar29** [CrocoBlock/suggestions#6045]
I have a sql in advanced mode that shows me 120 results. I am trying to use the pagination with the option they put, but I am not able. I've checked the documentation and I don't see anything. There is any error ?

**Comment by bortraws** [comment]
I am forced to use an advanced SQL Query in the Query Builder to show all purchased Woocommerce products of current user in A Listing Grid. It doesn't seem to be possible to do this in any other way but an Advanced SQL Query.  

Unfortunately Pagination and Smart Filters don't seem to work with an advanced SQL Query in the Query builder. I'd really like to have a solution for that as well.

**Comment by JuanCar29** [comment]
It is a very necessary function.

**Comment by morosmo** [comment]
Yes, This is much needed feature and crocoblock has no tutorials or guides to use sql query if they can make tutorials that will be easy for us. Also in query builder jet reviews query not works properly.
Please team have a look on query builder because if we combine profile builder & Query builder these are 2 most powerful tool of all time for jetengine which make thems industry lead.
